addons/io_scene_gltf2_msfs/io/msfs_light.py

```diff
@@ -52,8 +53,9 @@ def create(gltf2_node, blender_node, blender_light, import_settings):
     def export(gltf2_object, blender_object):
         # First, clear all KHR_lights_punctual extensions from children. TODO: remove children?
         for child in gltf2_object.children:
-            if child.extensions and child.extensions.get("KHR_lights_punctual"):
-                child.extensions.pop("KHR_lights_punctual")
+            if child.extensions:
```

Comment:
Why didn't you keep the "and child.extensions.get("KHR_lights_punctual")" part from the original code? Is it because you don't check the returned value from pop()?

Reply:
It's because I faced a situation where the `child.extensions` wasn't a dictionary and caused an error on `get` call.